ui/main/src/app/services/card.service.ts

```diff
@@ -74,7 +74,16 @@ export class CardService {
 
 
     public initCardSubscription() {
-        this.getCardSubscription()
+
+        // The use of concatMap + timer is to avoid having the browser stuck when 
+        // a lot of card is arriving. (It permit the browser to execute 
```

Comment:
```suggestion
        // a lot of card is arriving. (It allows the browser to execute 
```

Reply:
Hehe good thing you put a comment... otherwise I can see us puzzling over this one for a while in a few months ;-)

Reply:
ok